presubmit-service/check_if_pr_can_cherry_pick.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+
+# The organization containing the repo to be checked against, e.g. angular
+ORG=$1
+# The repo to be checked against, e.g. material2
+REPO=$2
+# The PR number to retrieve the patch files for, e.g. 1234
+PR=$3;
+# The branch in the repo to check for cherry-picking onto, e.g. 6.1.x
+BRANCH=$4;
+# The fullname of the repository to be checked against, e.g. angular/material2
+FULL_REPO="$ORG/$REPO"
+# Whether the PR can be cherry-picked on to the targetted branch.
+RESULT=false;
+
+# If the needed tmp directories don't exist, create them.
+[ ! -d "/tmp/presubmit_service/$FULL_REPO/" ] && mkdir -p /tmp/presubmit_service/$FULL_REPO;
```

Comment:
Are these checks necessary? Won't `mkrdir -p` no-op if the directory already exists?

Reply:
it safely fails, however it does fails and echo into `stderr`